### PR TITLE
Feature zoom to marker

### LINKED ISSUE #66: [FEATURE] Zoom to person for 10s
### Description

When a new event comes in, it would be great to zoom in to the map a bit so the persons pin can be seen better. Then after a certain time, zoom back out to show the whole map again

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by amandamartin-dev**
## Fixes Issue

Closes #66 

## Changes proposed

This will add a zoom to feature on each new event to highlight contributors

Another ticket will be opened to refine this feature to consider marker state and only zoom in on new to the map users

This feature will need testing in a live environment to see how it handles new events coming in and the pacing.

## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Screenshots

Short video of the feature in action
https://user-images.githubusercontent.com/97615019/206065582-2c7bcf7d-dd7c-4850-8feb-44afa2c09929.mov

## Note to reviewers

Needs to be tested with more events coming in at the expected live stream pace and likely refined based on that info

### CODE REVIEW

File: src/components/zoom-to-marker.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { useMap } from "react-leaflet"
+
+export default function PanToMarker(position) {
+    const map = useMap()
+    const currentLocation = map.getCenter()
+    const location = position.position ? position.position : [51.505, -0.09]
+
+        map.flyTo( location,10 );
```
Comment (eddiejaoude) [inline src/components/zoom-to-marker.js:8]: I prefer lines ending in `;`, but other lines are not - so we are consistent maybe we need an issue for a linter?

File: src/components/zoom-to-marker.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+        setTimeout(() => {
+            map.flyTo( [51.505, -0.09],3 )
```
Comment (eddiejaoude) [inline src/components/zoom-to-marker.js:10]: I see these 2 numbers are used twice, maybe they could be moved to a constant for reuse?

**Comment by eddiejaoude** [comment]
Looks great Amanda 👍

File: src/components/zoom-to-marker.js (hunk 2 above)
Comment (amandamartin-dev) [reply src/components/zoom-to-marker.js:10]: Ack! That was my test data.  I should have stopped playing wiht this earlier. Good catch!

**Review by eddiejaoude — APPROVED** [review]
LGTM 👍

**Pull request merged**